doc/lsst.meas.extensions.shapeHSM/hsm-moments.rst

```diff
@@ -0,0 +1,16 @@
+.. py:currentmodule:: lsst.meas.extensions.shapeHSM
+
+HSMMoments Plugins
+==================
+
+The purpose of the `HsmMoments` algorithms is to measure shapes of both the sources (or objects) and PSFs.
+The plugins wrap the implementation of the adaptive moments algorithm described in `Bernstein & Jarvis (2002) <http://adsabs.harvard.edu/abs/2002AJ....123..583B>`_ from `GalSim <https://github.com/GalSim-developers/GalSim>`_.
+This implementation is known to be more robust than :py:class:`SdssShape <lsst.meas.base.SdssShapeAlgorithm`> algorithm.
```

Comment:
In my single package docs build, this doesn't show up since it's referring to another package. But for the full pipelines build, I expect this to build well, or at least not crash the documentation build.

Reply:
You can always test it with a full build: https://developer.lsst.io/stack/building-pipelines-lsst-io-locally.html

Reply:
Oh, and this has a typo:
```suggestion
This implementation is known to be more robust than :py:class:`SdssShape <lsst.meas.base.SdssShapeAlgorithm>` algorithm.
```